mmf/models/base_model.py

```diff
@@ -224,7 +224,7 @@ def from_pretrained(cls, model_name, *args, **kwargs):
         instance = cls(config)
         instance.is_pretrained = True
         instance.build()
-        instance.load_state_dict(checkpoint)
+        instance.load_state_dict(checkpoint, strict=False)
```

Comment:
We should probably print like we did in other checkpoint case on what was unexpected and what was missing.